ssm/instance/types.go

```diff
@@ -22,24 +22,25 @@ type InstanceInfo struct {
 	Region     string
 	Profile    string
 	Tags       map[string]string
+	Attributes map[string]string
```

Comment:
This is the largest change. The addition of a `Attributes` field for adding non-tag instance attributes. I'm not sure if we should maintain the distinction of the two. Or simply just merge the two in favor of treating the whole `ec2.Instance`, tags included, as a struct of attributes we can display.

Treating them as separate entities is the least major approach and should be, non-breaking, but I'd love to hear feedback on this.